src/platform/backends/qemu/qemu_vm_process_spec.cpp

```diff
@@ -38,27 +38,27 @@ QStringList initial_qemu_arguments(const mp::VirtualMachineDescription& desc, co
     auto mem_size = QString::number(desc.mem_size.in_megabytes()) + 'M'; /* flooring here; format documented in
     `man qemu-system`, under `-m` option; including suffix to avoid relying on default unit */
 
-    QStringList args{"--enable-kvm",
-                     "-hda",
-                     desc.image.image_path,
-                     "-smp",
-                     QString::number(desc.num_cores),
-                     "-m",
-                     mem_size,
-                     "-device",
-                     QString("virtio-net-pci,netdev=hostnet0,id=net0,mac=%1")
-                         .arg(QString::fromStdString(desc.default_interface.mac_address)),
-                     "-netdev",
-                     QString("tap,id=hostnet0,ifname=%1,script=no,downscript=no").arg(tap_device_name),
-                     "-qmp",
-                     "stdio",
-                     "-cpu",
-                     "host",
-                     "-chardev",
-                     "null,id=char0",
-                     "-serial",
-                     "chardev:char0",
-                     "-nographic"};
+    QStringList args{
+        "--enable-kvm",
+        "-hda",
+        desc.image.image_path,
+        "-smp",
+        QString::number(desc.num_cores),
+        "-m",
+        mem_size,
+        "-device",
+        QString("virtio-net-pci,netdev=hostnet0,id=net0,mac=%1").arg(QString::fromStdString(desc.default_mac_address)),
+        "-netdev",
+        QString("tap,id=hostnet0,ifname=%1,script=no,downscript=no").arg(tap_device_name),
+        "-qmp",
+        "stdio",
+        "-cpu",
+        "host",
+        "-chardev",
+        "null,id=char0",
+        "-serial",
+        "chardev:char0",
+        "-nographic"};
```

Comment:
Did you force this reformatting? It seems to me that clang-format did a mistake.

Reply:
I have a git hook to run clang-format on my commits and refuse the commit if changes are needed. That hook signaled this, and running manually, this was the result. 

Clang-format's decisions are not always the most pleasing, but it's just following rules. I think here it prefers to save one level of indentation (in line 50).

Reply:
If this bothers you, I can create a separate var for the big virtio string.

Reply:
No, it doesn't bother me at all. Just commenting.